### PR TITLE
XEP-0198: added optional h attribute to failed element on sm resumption

### PR DESCRIPTION
**Pull request by iNPUTmice**
as previously discussed on the mailing list: http://mail.jabber.org/pipermail/standards/2015-May/029851.html

### CODE REVIEW

**Comment by Flowdalic** [comment]
But the schema change is missing. It must be adopted to the new `h` attribute of the `failed` element.

**Comment by iNPUTmice** [comment]
> It must be adopted to the new h attribute of the failed element.

done. that should be verified though since I have literally no idea of what I'm doing. Guess I was sick the day we had XML schemas in school.

File: xep-0198.xml [hunk 1]
```diff
@@ -574,6 +591,9 @@ S: <a h='10'/>
                    maxOccurs='1'>
         <xs:group ref='err:stanzaErrorGroup'/>
       </xs:sequence>
+      <xs:attribute name='h' 
+                    type='xs:integer'
```
Comment (Flowdalic) [inline xep-0198.xml]: Should be `xs:unsignedInt`.

**Comment by iNPUTmice** [comment]
changed schema to use unsignedInt and squashed

**Comment by dwd** [comment]
This looks good; I'll raise this on the list.

**Comment by iNPUTmice** [comment]
Can I bump this issue? I think there is already a thread on the standards lists that predates this PR. - If I recall people couldn't agree on whether or not this needs a namespace bump.

**Comment by stpeter** [comment]
I think this change is fine.

**Comment by dwd** [comment]
I thought we could get away without, too.

**Comment by iNPUTmice** [comment]
So given the council meeting from March 23rd 2016 and the author approval from 2 hours ago this can be merged I assume?

**Comment by stpeter** [comment]
Yes, I think so!

**Comment by iNPUTmice** [comment]
>> So given the council meeting from March 23rd 2016 and the author approval from 2 hours ago this can be merged I assume?

> Yes, I think so!

So can someone please press the merge button then? (@SamWhited maybe?)

**Comment by SamWhited** [comment]
I can press the merge button, but I have yet to figure out how to update XEPs on the website. I suppose I could go ahead and merge and open another issue to update the site, but it seems like we might as well wait until someone's around who can do both.

**Comment by ashward** [comment]
Hi All. Sorry I haven't had a chance to look at this - I've been moving house so have been off-grid. I'll try and sort this out today.

**Comment by stpeter** [comment]
@SamWhited Let's figure out how to get you access (if that's the issue) and then document how it all works so that you, me, and @ashward can tag-team on things.

**Comment by SamWhited** [comment]
@stpeter I have access, the instructions in the editors readme just weren't right and I didn't spend any time trying to figure it out; we should definitely sit down and upgrade those one of these days (or just setting up a CI system or something to do it for us).

**Comment by ashward** [comment]
1.5 has been released.

**Pull request closed without merging**